### PR TITLE
Screen Home

### PR DESCRIPTION
**Pull request by CrypeDJ**

### CODE REVIEW

File: app/src/main/java/com/crype/bankapp/presentation/components/UIComponents.kt [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                date = date,
+                transactionProgress = transactionProgress,
+                money = money)
+        }
+    }
+}
+
+@Composable
+fun TransactionInfo (
+    senderName: String,
```
Comment (Razor0744) [inline app/src/main/java/com/crype/bankapp/presentation/components/UIComponents.kt]: It should be in another file

File: app/src/main/java/com/crype/bankapp/presentation/screen/HomeScreen.kt [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+
+
+
+
+
+@Composable
+fun AccountName(
+    accountName: String,
+    numberOfAccount:String,
```
Comment (Razor0744) [inline app/src/main/java/com/crype/bankapp/presentation/screen/HomeScreen.kt]: the same, another file

File: app/src/main/java/com/crype/bankapp/MainActivity.kt [hunk 3]
```diff
@@ -1,47 +1,51 @@
 package com.crype.bankapp
```
Comment (AnnSlizh) [inline app/src/main/java/com/crype/bankapp/MainActivity.kt:2]: бла бла бла

File: app/src/main/java/com/crype/bankapp/presentation/screen/HomeScreen.kt (hunk 2 above)
Comment (CrypeDJ) [reply app/src/main/java/com/crype/bankapp/presentation/screen/HomeScreen.kt]: got it

File: app/src/main/java/com/crype/bankapp/presentation/components/UIComponents.kt (hunk 1 above)
Comment (CrypeDJ) [reply app/src/main/java/com/crype/bankapp/presentation/components/UIComponents.kt]: got it

**Pull request merged**